include/deal.II/differentiation/ad/adolc_number_types.h

```diff
@@ -17,8 +17,26 @@
 
 #include <deal.II/base/config.h>
 
+#include <deal.II/base/exceptions.h>
+#include <deal.II/base/numbers.h>
+
+#include <deal.II/differentiation/ad/ad_number_traits.h>
+#include <deal.II/differentiation/ad/ad_number_types.h>
+
 #include <type_traits>
```

Comment:
Could you move this down to be with the other standard header?

Reply:
Certainly -- so fixed!